content/04.results.md

```diff
@@ -13,15 +13,39 @@ Stub for the process for contributing analytical code (*Figure {@fig:FigS1}S1* a
+
+#### Embryonal tumors (N = 131)
+*Figure {@fig:Fig2}2B* depicts the oncoprint for 131 primary embryonal tumors within the OpenPBTA. 
+The majority (N = 97) are medulloblastomas and span the spectrum of molecular subtypes: WNT, SHH, Group3, and Group 4 (see *Molecular Subtyping of CNS Tumors*) as well as mutational landscape. 
+We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].
+
+#### Diffuse astrocytic and oligodendroglial tumors
+In *Figure {@fig:Fig2}2C*, we show an oncoprint for diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (N = 29) biopsied at diagnosis. The single oligodendroglioma sample in the OpenPBTA does not contain mutations in the genes shown and is therefore not present in this oncoprint. 
+We found _TP53_ (57%, X/64) and _H3F3A_ (52%, X/64) to be the most mutated and co-occurring genes (*Figure {@fig:Fig2}3A*), followed by frequent mutations in _ATRX_ (30% N/64). 
+We found recurrent amplifications and fusions in _EGFR_, _MET_, _PDGFRA_, and _KIT_, highlighting that these tumors utilize multiple oncogenic mechanisms to activate tyrosine kinases, as has been previously reported [@doi:10.1007/s12254-021-00680-x; @doi:10.1002/ijc.32258; @doi:10.1016/j.ccell.2017.08.017; @doi:10.1186/s40478-020-00905-w]. 
+The two tumors with ultra-high tumor mutation burden (TMB) (> 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].
+Gene set enrichment analysis showed upregulation (ANOVA p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (*Figure {@fig:Fig2}4B*).
+
+#### Other CNS tumors (N = 199)
+*Figure {@fig:Fig2}2D* depicts an oncoprint for the remaining primary CNS tumors within OpenPBTA. 
+We observed 8% (X/131) of tumors to be C11orf95::RELA fusion positive ependymomas and 12% (X/131) to be adamantinomatous craniopharayngiomas driven by mutations in _CTNNB1_. 
+Multiple cancer types contained somatic mutations or fusions in _NF2_ (7%, comprised of ependymomas, desmoplastic infantile astrocytoma and ganglioglioma, …, and …) and _BRAF_ (4%), in addition to rare ependymoma fusions (< 4% of all embryonal tumors) in _YAP1_, _KRAS_, _ERBB4_, and _MYB_. 
+The majority of dysembryoplastic tumors harbored alterations in _FGFR1_, including _FGFR1::TACC1_ fusions [@doi:10.1007/s00401-016-1539-z; @doi:10.1038/s41525-017-0014-7; @doi:10.3171/2019.8.JNS191266; @doi:10.1007/s11060-020-03509-5; @doi:10.1016/j.jaad.2017.05.059; @doi:10.1016/j.ccell.2015.04.002; @doi:10.1186/s40478-020-01056-8; @doi:10.1007/s00381-017-3481-3].
```

Comment:
Just noting this is a very large number of citations for a pretty simple statement!